Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -76,8 +77,7 @@ class ContentBlockerHelper {
     init(tab: Tab, profile: Profile) {
         self.ruleStore = WKContentRuleListStore.default()
         if self.ruleStore == nil {
-            print("WKContentRuleListStore unavailable.")
-            assert(false)
+            assert(false, "WKContentRuleListStore unavailable.")
```

Comment:
WKContentRuleListStore unavailability is unknown as to when/why this happens, in the release case, I want to return and no-op any further work.